game-backend/src/game-engine/food-spawner/spawnFood.ts

```diff
@@ -0,0 +1,19 @@
+import hyperid from 'hyperid';
+import { ARENA_WIDTH, ARENA_HEIGHT } from '../../../../darwin-types/Arena';
+import { Food } from '../../../../darwin-types/game-objects/Food';
+
+const generateId = hyperid();
+
+const spawnFood = (): Food => {
+  const food: Food = {
+    id: generateId(),
+    type: 'food',
+    position: {
+      x: Math.floor(Math.random() * ARENA_WIDTH),
+      y: Math.floor(Math.random() * ARENA_HEIGHT),
+    },
```

Comment:
This could theoretically spawn food on a unit. We need something that handles such cases.

Reply:
In progress, also for the unit spawner, but not in this branch